packages/ui/src/i18n/dictionaries/authenticator/rs.ts

```diff
@@ -0,0 +1,51 @@
+export const ruDict = {
```

Comment:
Ooh one more thing, this file name should be `ru.ts` for imports to work correctly.

Reply:
Done